src/plugindata.h

```diff
@@ -183,6 +183,11 @@ typedef struct GeanyData
 	struct GeanyTemplatePrefs	*template_prefs;	/**< Template settings */
 	gpointer					*_compat;			/* Remove field on next ABI break (abi-todo) */
 	GSList						*filetypes_by_title; /**< See filetypes.h#filetypes_by_title. */
+	/** Global object signalling events via signals
+	 *
+	 * This is mostly for language bindings. Otherwise prefer to use
+	 * plugin_signal_connect() instead this which adds automatic cleanup. */
+	GeanyObject * const			object;
```

Comment:
`const` is cute and all but mostly useless (if anything changes in that structure I prefer not imagine which hells break loose), and is both different from the rest and disallows you to fill the structure. Meh.